app/javascript/dashboard/routes/dashboard/settings/inbox/components/AccountHealth.vue

```diff
@@ -0,0 +1,226 @@
+    const whatsappBusinessUrl = `https://business.facebook.com/latest/whatsapp_manager/phone_numbers/?business_id=${businessId}&tab=phone-numbers`;
+    window.open(whatsappBusinessUrl, '_blank');
+  } else {
+    // Fallback to general WhatsApp Business Manager if business_id is not available
+    const fallbackUrl = 'https://business.facebook.com/';
+    window.open(fallbackUrl, '_blank');
+  }
+};
+
+const getQualityRatingTextColor = rating =>
+  QUALITY_COLORS[rating] || QUALITY_COLORS.UNKNOWN;
+
+const formatTierDisplay = tier =>
+  t(`INBOX_MGMT.ACCOUNT_HEALTH.VALUES.TIERS.${tier}`) || tier;
+
+const formatStatusDisplay = status =>
+  t(`INBOX_MGMT.ACCOUNT_HEALTH.VALUES.STATUSES.${status}`) || status;
+
+const formatModeDisplay = mode =>
+  t(`INBOX_MGMT.ACCOUNT_HEALTH.VALUES.MODES.${mode}`) || mode;
+
+const getModeStatusTextColor = mode => MODE_COLORS[mode] || 'text-n-slate-12';
+
+const getStatusTextColor = status => STATUS_COLORS[status] || 'text-n-slate-12';
+</script>
+
+<template>
+  <div class="gap-4 pt-8 mx-8">
+    <div
+      class="px-5 py-5 space-y-6 rounded-xl border shadow-sm border-n-weak bg-n-solid-2"
+    >
+      <div
+        class="flex flex-col gap-5 justify-between items-start w-full md:flex-row"
+      >
+        <div>
+          <span class="text-base font-medium text-n-slate-12">
+            {{ t('INBOX_MGMT.ACCOUNT_HEALTH.TITLE') }}
+          </span>
+          <p class="mt-1 text-sm text-n-slate-11">
+            {{ t('INBOX_MGMT.ACCOUNT_HEALTH.DESCRIPTION') }}
+          </p>
+        </div>
+        <ButtonV4
+          sm
+          solid
+          blue
+          class="flex-shrink-0"
+          @click="handleGoToSettings"
+        >
+          {{ t('INBOX_MGMT.ACCOUNT_HEALTH.GO_TO_SETTINGS') }}
+        </ButtonV4>
+      </div>
+
+      <div v-if="healthData" class="grid grid-cols-1 gap-4 xs:grid-cols-2">
+        <div
+          v-for="item in healthItems"
+          :key="item.key"
+          class="flex flex-col gap-2 p-4 rounded-lg border border-n-weak bg-n-solid-1"
+        >
+          <div class="flex gap-2 items-center">
+            <span class="text-sm font-medium text-n-slate-11">
+              {{ item.label }}
+            </span>
+            <i
+              v-tooltip.top="item.tooltip"
+              class="flex-shrink-0 w-4 h-4 cursor-help i-lucide-info text-n-slate-9"
+            />
+          </div>
+          <div class="flex items-center">
+            <span
+              v-if="item.type === 'quality'"
+              class="inline-flex items-center px-2 py-0.5 h-6 text-xs font-medium rounded-md bg-n-alpha-2"
+              :class="getQualityRatingTextColor(item.value)"
+            >
+              {{ item.value }}
+            </span>
+            <span
+              v-else-if="item.type === 'status'"
+              class="inline-flex items-center px-2 py-0.5 h-6 text-xs font-medium rounded-md bg-n-alpha-2"
```

Comment:
https://github.com/chatwoot/chatwoot/pull/12556/files#r2394125933
```suggestion
              class="inline-flex items-center px-2 py-0.5 min-h-6 text-xs font-medium rounded-md bg-n-alpha-2"
```